component/core/src/main/resources/db/changelog/social-rdbms.db.changelog-1.0.0.xml

```diff
@@ -914,4 +914,11 @@
         </sql>
     </changeSet>
 
+    <changeSet author="social" id="1.0.0-93">
+        <addColumn tableName="SOC_PROFILE_PROPERTY_SETTING">
+            <column name="IS_HIDDENABLE" type="BOOLEAN" defaultValue="false">
```

Comment:
shouldn't this be true as default value ?

Reply:
I don't know,
it is a functional question